docs/project/changelog.md

```diff
@@ -399,7 +399,7 @@ async function runPythonAsync(code, globals) {
   {pr}`2028`
 
 - New packages: logbook {pr}`1920`, pyb2d {pr}`1968`, and threadpoolctl (a
-  dependency of scikit-learn) {pr}`2065`
+  dependency of scikit-learn) {pr}`2065`, libhdf5 {pr}`1799`, h5py {pr}`2180`
```

Comment:
Please move this up to the "Unreleased" section (it's ended up in the v0.20.0 changelog which isn't correct anymore).